api/Parser.cpp

```diff
@@ -357,53 +357,40 @@ EndpointDescription parsePatchEndpoint(const nlohmann::json& data, const std::st
         {
-            videoChannel._transport.set(parsePatchEndpointTransport(videoJson["transport"]));
-        }
-
-        for (const auto& ssrcJson : optionalJsonArray(videoJson, "ssrcs"))
-        {
-            const auto ssrc = ssrcJson.is_string() ? std::stoul(ssrcJson.get<std::string>()) : ssrcJson.get<uint32_t>();
-            videoChannel._ssrcs.push_back(ssrc);
+            videoChannel.transport.set(parsePatchEndpointTransport(videoJson["transport"]));
         }
 
         for (const auto& payloadTypeJson : optionalJsonArray(videoJson, "payload-types"))
         {
-            videoChannel._payloadTypes.emplace_back(parsePatchEndpointPayloadType(payloadTypeJson));
+            videoChannel.payloadTypes.emplace_back(parsePatchEndpointPayloadType(payloadTypeJson));
         }
 
         for (const auto& rtpHdrExtJson : optionalJsonArray(videoJson, "rtp-hdrexts"))
         {
             const auto id = rtpHdrExtJson["id"].get<uint32_t>();
             if (id > 0 && id < 15)
             {
-                videoChannel._rtpHeaderExtensions.emplace_back(id, rtpHdrExtJson["uri"].get<std::string>());
-            }
-        }
-
-        for (const auto& ssrcGroupJson : optionalJsonArray(videoJson, "ssrc-groups"))
-        {
-            api::EndpointDescription::SsrcGroup ssrcGroup;
-            for (const auto& ssrcJson : requiredJsonArray(ssrcGroupJson, "ssrcs"))
-            {
-                const uint32_t ssrc =
-                    ssrcJson.is_string() ? std::stoul(ssrcJson.get<std::string>()) : ssrcJson.get<uint32_t>();
-                ssrcGroup._ssrcs.push_back(ssrc);
+                videoChannel.rtpHeaderExtensions.emplace_back(id, rtpHdrExtJson["uri"].get<std::string>());
             }
-            ssrcGroup._semantics = ssrcGroupJson["semantics"].get<std::string>();
-            videoChannel._ssrcGroups.emplace_back(std::move(ssrcGroup));
         }
 
-        for (const auto& ssrcAttributeJson : optionalJsonArray(videoJson, "ssrc-attributes"))
+        for (const auto& stream : optionalJsonArray(videoJson, "streams"))
         {
-            api::EndpointDescription::SsrcAttribute ssrcAttribute;
-            ssrcAttribute._content = ssrcAttributeJson["content"].get<std::string>();
-            for (const auto& ssrcJson : requiredJsonArray(ssrcAttributeJson, "ssrcs"))
+            videoChannel.streams.emplace_back();
+            auto& videoStream = videoChannel.streams.back();
+            for (const auto& rtpSource : requiredJsonArray(stream, "sources"))
             {
-                const auto ssrc =
-                    ssrcJson.is_string() ? std::stoul(ssrcJson.get<std::string>()) : ssrcJson.get<uint32_t>();
-                ssrcAttribute._ssrcs.push_back(ssrc);
+                api::EndpointDescription::VideoStream::Level level;
+                const auto& main = rtpSource["main"];
+                level.main = (main.is_string() ? std::stoul(main.get<std::string>()) : main.get<uint32_t>());
```

Comment:
Should we really support this. We specify the API and there is no point in doing ssrc as string, and no examples of it.